.circleci/config.yml

```diff
@@ -439,9 +439,11 @@ jobs:
       - run:
           name: foundryup
           command: |
+            curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y
+            export CARGO_NET_GIT_FETCH_WITH_CLI=true
             curl -L https://foundry.paradigm.xyz | bash
             source $HOME/.bashrc
-            foundryup
+            foundryup --pr 2634
```

Comment:
They merged the PR, maybe we can stick to the old approach. Although freezing foundry to a stable version may still be useful.